src/resource/ubridge.c

```diff
@@ -3434,10 +3434,6 @@ static int _cmd_handler(sid_resource_event_source_t *es, void *data)
 		return -1;
 	}
 
-	if ((r = _export_kv_store(cmd_res)) < 0) {
```

Comment:
@prajnoha am I missing something here?  It looks like we append the kv store even when responding to commands other than "dump"?

Reply:
Ah yes, this is something I've also noticed recently - it's actually a bug!

The `_export_kv_store` is supposed to export the KV store snapshot diffs from SID worker process and export it to main process for syncing with main DB. Of course, this (currently) makes sense only for the `usid scan` command and we shouldn't be calling `_export_kv_store` for any other cases, including any of `sidctl` commands, because they do not change the database in any way, just peeking at it.

I have a patch for this in my queue, will commit that...

Reply:
...well, for records marked as belonging to `KV_NS_UDEV` namespace, the `_export_kv_store` exports to udev db actually, not the SID db. But all the other namespaces (`KV_NS_DEVICE`, `KV_NS_MODULE`, `KV_NS_GLOBAL`) do sync with main SID db. So from this perspective, the SID db is extension to udev db in a way.

Reply:
Ok - thanks for confirming.  It causes problems with the JSON response.  When sidctl gets the response, the size includes the data that is added via _export_kv_store().

Reply:
This should be patched within PR #96. Please try with that patch...

Reply:
Ok - will test now.

Reply:
Hold on.... 

I made a mistake testing and ran an old version (or something).  It is working.